app/utils/ControllerHelper.scala

```diff
@@ -43,13 +44,21 @@ object ControllerHelper {
   def hasIncompleteResponsiblePerson(rps: Option[Seq[ResponsiblePerson]]): Boolean =
     rps.exists((data: Seq[ResponsiblePerson]) => filter(data).exists(_.isComplete equals false))
 
-  def hasInvalidRedressScheme(eabOpt: Option[EstateAgentBusiness]): Boolean =
+  //TODO - can be removed when we remove the old EAB models
+  def hasInvalidRedressScheme(eabOpt: Option[EstateAgentBusiness]): Boolean = {
     eabOpt.exists(eab =>
       eab.redressScheme.exists(redressScheme =>
         redressScheme match {
           case OmbudsmanServices | Other(_) => true
           case _  => false
         }))
+  }
+  def hasInvalidRedressSchemeNewEab(eabOpt: Option[Eab]): Boolean = {
```

Comment:
please separate  method definitions adding blank line between 55-56

Reply:
When overriding a method that will soon be removed I always leave no gap. But I can add a gap if it makes you happy. 👍

Reply:
right if that's the case let's leave it